README.md

```diff
@@ -79,6 +79,32 @@ The `doc` target builds the API documentation locally:
 
     make doc
 
+## Docker
+
+The source ships with the convenience script `docker_build.sh`, which will
+create the Docker images and save them as `tar.gz` archives (when invoked
+without arguments).
+
+To run the container, you need to provide a volume to the mountpoint `/data`:
+
+The default command will print the help message
+
+``` sh
+docker run -v /tmp/vast:/data vast-io/vast
+```
+
+Use detach and publish the default port to start a VAST node
+
+``` sh
+docker run -d -p 42000:42000 -v /tmp/vast:/data vast-io/vast start
+```
+
+Import a Zeek conn log to the detached server instance
+
+``` sh
+cat zeek_conn.log | docker run -i -v /tmp/vast:/data vast-io/vast -e '172.17.0.2' import zeek
```

Comment:
I think it's helpful to explain that `172.17.0.2` belongs to the VAST node started previously.

Reply:
The example is updated. We now assign an explicit ip to the vast_node container.